### DISCUSSION TITLE
Auto ID not incremeting any more

### DISCUSSION THREAD

**Discussion by JohannesM007** [Budibase/budibase#10585] [Help] [answered]
Since today 15-05-2023 the Auto ID of the Budibase DB tables are not incrementing any more.
We use Couche DB and a local hosted version 2.6.9 and also tried the latest version 2.6.11
If i add a new row in any of my tables the ID will not increment.
We tried to delete the last data record but this also will not help.
We tried Google Chrome and Firefox as well.
![image](https://github.com/Budibase/budibase/assets/125383022/75f5eeac-07bc-4cf4-8705-b81aea0e4b1f)
Whats going wrong?  Is there a known issue in the moment?
Regards
Johannes

**Comment by melohagan** [comment, marked as answer]
Hi @JohannesM007 

Thank you for raising this - I've brought it up with the dev team and logged the bug here: https://github.com/Budibase/budibase/issues/10599

**Reply by JohannesM007** [reply]
Thank you @melohagan!
It's working again!
Any idea how i can fix the wrong Auto IDs?

**Reply by melohagan** [reply]
Hey @JohannesM007 

You may be able to use the public API to update the wrong Auto IDs using the **_id**. I'm not certain if this will work though.  https://docs.budibase.com/reference/put_tables-tableid-rows-rowid

**Reply by JohannesM007** [reply]
Thank you @melohagan for the tip with the API solution!
The AutoID's are now up to date again ;-)
It was a manual effort but we were able to correct the continuous counting.
Regards Johannes
